VEDA_OS01/views.py

```diff
@@ -336,15 +336,12 @@ def _ingest_from_s3_bucket(self, request_body):
                     'Returning 200 to avoid retry attempts.'.format(
                         vd_key=vd_key
                 ))
-                successful_ingest = 200
             else:
-                file_discovery = FileDiscovery()
-                file_discovery.bucket = bucket
-                successful_ingest = file_discovery.validate_metadata_and_feed_to_ingest(
-                    video_s3_key=vd_key
+                validate_metadata_and_feed_to_ingest(
+                    video_s3_key=vd_key,
+                    bucket=bucket
                 )
-
-            status = 200 if successful_ingest else 400
```

Comment:
Are we sure we no longer needs 400 error code?

Reply:
I thought we didn't, but validate_metadata_and_feed_to_ingest still returns a False in one case, so yeah we might want it.